packages/dual-channel/src/app/components/embedded-items/index.js

```diff
@@ -139,7 +139,18 @@ const ItemAnimationWrapper = styled.div`
   ${mq.tabletBelow`
     text-align: center;
     background-color: #f1f1f1;
+
+    img {
```

Comment:
This will only apply to embedded items which are images. (It does not deal with infogram, video...etc). So it's better to document here that this is a temporary workaround.